pkgs/tools/misc/cpufetch/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "cpufetch";
+  version = "0.94";
+
+  src = fetchFromGitHub {
+    owner  = "Dr-Noob";
+    repo   = "cpufetch";
+    rev    = "v${version}";
+    sha256 = "1gncgkhqd8bnz254qa30yyl10qm28dwx6aif0dwrj38z5ql40ck9";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    install -Dm755 cpufetch   $out/usr/bin/cpufetch
```

```suggestion
    install -Dm755 cpufetch   $out/bin/cpufetch
```

otherwise nixos will not find it by default.

---

Thanks, I wasn't sure if I should keep the upstream path or not.